cumulusci.yml

```diff
@@ -126,37 +131,25 @@ flows:
                 task: deploy_dev_config
 
     config_managed:
+        description: Configure an org for use as a dev org after package metadata is deployed
         steps:
             2:
-                # Configure update_admin_profile
-                options:
-                    managed: True
-            3:
                 task: deploy_dev_config
                 options:
                     unmanaged: False
 
-    config_packaging:
-        steps:
-            1:
-                # Configure update_admin_profile
-                options:
-                    managed: True
-                    namespaced_org: True
-                    skip_record_types: True
-
     config_qa:
+        descripton: Configure an org for use as a QA org after package metadata is deployed
         steps:
             3:
                 task: execute_install_apex
             4:
-                task: deploy_dev_config
```

Comment:
@jstvz do you think it is ok for us to remove this line? config_qa should only deploy qa config?

Reply:
There's some overlap here, but I lean towards deploying it as part of the QA config. This is for two reasons: first, because new schema will often be updated here first, and second, to avoid having to duplicate sets of metadata.

That said, why did you want to remove it?

Reply:
@jstvz that makes a lot of sense. I was under the impression that dev config is only used for dev org and qa org should use its own config. I will go ahead and add it back.